docs/source/reference/visualization/index.rst

```diff
@@ -21,3 +21,11 @@ optuna.visualization
    optuna.visualization.plot_param_importances
    optuna.visualization.plot_slice
    optuna.visualization.is_available
+
+.. note::
+    The following :mod:`~optuna.visualization.matplotlib` module uses matplotlib as a backend.
```

Comment:
Not strong opinion, but the [official site](https://matplotlib.org/) uses `Matplotlib` instead of `matplotlib`.

```suggestion
    The following :mod:`~optuna.visualization.matplotlib` module uses Matplotlib as a backend.
```